packages/trader/src/Modules/CFD/Containers/compare-accounts-modal.jsx

```diff
@@ -196,7 +196,7 @@ const getAccounts = ({ landing_companies, platform, is_logged_in }) => {
                 financial: localize('75%'),
```

Comment:
Do we need to update `financial` stop out level?

Reply:
yeah, fixed